lib/Room.ts

```diff
@@ -75,7 +75,7 @@ export const updateRoomLogData = async (rid: string, data: IRoomCustomField, rea
 };
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const closeChat = async (modify: IModify, read: IRead, rid: string, _persistence?: IPersistence) => {
+export const closeChat = async (modify: IModify, read: IRead, rid: string, app: IApp, closeChatDesc?: string, _persistence?: IPersistence) => {
```

Comment:
Can we remove `_persistence` and lint comment, if it's not being used anywhere? also